zetasql/fuzzing/zetasql_expression_fuzzer.cc

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <iostream>
+
+#include "libprotobuf_mutator/src/libfuzzer/libfuzzer_macro.h"
+#include "zetasql/public/evaluator.h"
+#include "zetasql/fuzzing/oss_fuzz.h"
+#include "zetasql/fuzzing/zetasql_expression_grammar.pb.h"
+#include "zetasql/fuzzing/zetasql_expression_proto_to_string.h"
+
+using namespace zetasql_expression_grammar;
```

Comment:
Per the Google style guide, using namespace statements should be avoided.

Instead, consider using specific class names (e.x. using zetasql_expression_grammer::Expression).

Reply:
So the convention was assumed from SQLite: https://chromium.googlesource.com/chromium/src/third_party/+/refs/heads/master/sqlite/fuzz/sql_query_proto_to_string.cc

It seems that we need to primarily/extensively use functions generated by protobuf. In an ever growing file that depends on various declared protobuf messages, declaring "using" for each class may unnecessarily bloat the codebase and wouldn't help much in readability.

Reply:
I believe the sqlite code is actually part of the sqlite project.  They probably have a different (potentially conflicting!) style guide.  :)

So the current guidance is here:  https://google.github.io/styleguide/cppguide.html#Namespaces and explicitly forbids "using namespace foo" as it pollutes the namespace.  This is generally a maintainability concern--if you have a long file and one "using namespace ..." and you hit a name collision, you have a LOT of changes/cleanup/etc. to do to fix the problem.  (This gets even worse in headers.)

Frankly, this isn't a rule I like all that much for .cc files (I think it makes a lot of sense for headers).  You will often see a whole slew of "using namespace1::namespace2::class1; using namespace1::namespace2::class2; ..." in various Google code because of it, though, and I'd lean towards changing this, simple on the "when in Rome..." philosophy.  :)